ui/src/app/shared/services/requests.ts

```diff
@@ -39,28 +39,11 @@ export default {
         return auth(superagent.del(apiUrl(url)));
     },
 
-    loadEventSource(url: string, allowAutoRetry = false): Observable<string> {
+    loadEventSource(url: string): Observable<string> {
         return Observable.create((observer: Observer<any>) => {
             const eventSource = new EventSource(url);
-            let opened = false;
-            eventSource.onopen = () => {
```

Comment:
this is canonical way to do this, I hope is more robust, certainly surfaces errors correctly